src/k8s/cmd/k8s/k8s_kubectl.go

```diff
@@ -21,6 +21,10 @@ var (
 			if os.Getenv("KUBECONFIG") == "" {
 				os.Setenv("KUBECONFIG", "/etc/kubernetes/admin.conf")
 			}
+			// Set a default editor that comes with the snap so that 'kubectl edit' works
+			if os.Getenv("EDITOR") == "" {
+				os.Setenv("EDITOR", "nano")
```

Comment:
Do we need to pack this with the snap in strict confinement?

Reply:
We already do in https://github.com/canonical/k8s-snap/blob/main/snap/snapcraft.yaml#L148